src/typescript/sdk/tests/post-test.ts

```diff
@@ -2,5 +2,5 @@
 import { DockerTestHarness } from "../src/utils/test/docker/docker-test-harness";
 
 export default async function postTest() {
-  await DockerTestHarness.stop();
+  await DockerTestHarness.stop(false);
```

Comment:
I think we should make this a named variable or make the function name clearer here. I can't remember what the boolean value does when calling `stop`

It should probably just be two different functions with the functionality conveyed in the function name, and then remove the boolean arg. Easier to read that way